### ISSUE TITLE
Post Merge > All themes don't set text contrast on switching mode

### ISSUE THREAD

**Issue by cheenamalhotra** [microsoft/azuredatastudio#23521]
- Azure Data Studio Version: Insiders
- OS Version: Windows 11

Except Light Modern and Dark Modern, other theme texts lack contrast on switch from alternate mode.

Steps to repro dark theme issue:
1. Select Light Modern.
2. Go to Settings > Themes > Color Theme and directly select a dark theme from the list except "Dark Modern".
Notice the theme doesn't render all dashboard text readable:

![image](https://github.com/microsoft/azuredatastudio/assets/13396919/252fa506-9bf6-44d8-bdae-3b31c28f16a8)

If switching from Dark Modern to other dark themes, the contrast is set as expected.

Steps to repro light theme issue:
1. Select Dark Modern.
2. Go to Settings > Themes > Color Theme and directly select any light theme other than "Light Modern"

Notice the theme doesn't reset contrast on all dashboard text:
![image](https://github.com/microsoft/azuredatastudio/assets/13396919/27a20395-d265-411b-81a2-1370c74ed9a8)

If switching from Light Modern to other light themes, the contrast is set as expected.
